docs/7_modules.md

```diff
@@ -106,3 +106,446 @@
+                type: 'translatable'
+                type_options:
+                    type: 'text' #Field type
+                    type_options:
+                        attr:
+                                data-edit-content-input: 'readMoreButtonText' #This data attribute allows the button text to be previewed during editing.
+      
+            read_more_button_link:
+                type: 'symfonyRoute' #Field type: Symfony route selector
+                type_options:
+      
+            read_more_button_class:
+                type: 'class' #Field type
+                type_options:
+                    attr:
+                        data-edit-class-input: 'readMoreButtonClass' #This data attribute allows the button CSS classes to be previewed during editing.
+      
+             score:
+                type: 'choice' #Field type
+                type_options:
+                    choices:
+                        "rating.form.score.values.one": 1
+                        "rating.form.score.values.two": 2
+                        "rating.form.score.values.three": 3
+                        "rating.form.score.values.four": 4
+                        "rating.form.score.values.five": 5
+                    attr:
+                        data-edit-class-input: 'scoreClass'
+
+
+```
+
+##### translations/sfs_cms_modules.en.yaml
+
+```
+rating:
+    label: "Rating"
+    prototype_button: "Rating"
+    prototype_button_icon: '<i class="bi bi-star"></i>' #Icon in module selector modal
+    form:
+        locale_filter.label: "Locales" #If your application has languages, a language selector will appear to control the display of the module by language.
+        background.label: "Background image"
+        media.label: "Image"
+        id.label: "Id"
+        class.label: "Classes"
+        title_class.label: "Title classes"
+        title:
+            label: "Title"
+            placeholder: "Rating title"
+        description:
+            label: "Description"
+            placeholder: "Rating description"
+        read_more_button_text.label: "Button text"
+        read_more_button_link:
+            route_name.label: "Button link"
+            route_params.label: "Link parameters"
+        bg_color.label: "Background color"
+        read_more_button_class.label: "Button classes"
+        score.label: "Score"
+        score.values:
+            one: "1 star"
+            two: "2 stars"
+            three: "3 stars"
+            four: "4 stars"
+            five: "5 stars"
+            
+```
+##### form.yaml
+
+```
+<<div class="row">
+    <div class="col-12">
+        {{ form_row(form.id) }}
+    </div>
+    <div class="col-12">
+        {{ form_row(form.class) }}
+    </div>
+    <div class="col-12">
+        {{ form_row(form.media) }}
+    </div>
+    <div class="col-12">
+        {{ form_row(form.title) }}
+    </div>
+    <div class="col-12">
+        {{ form_row(form.title_type, {'attr': {'data-cms-module-title-type-field': ''}}) }}
```

Comment:
Why is not this data attribute set in config.yaml? It could work also work on here, but the rest of the inputs have their data attributes set on config.yaml.

Reply:
Changed!